code/game/objects/item_damage.dm

```diff
@@ -28,6 +28,16 @@
 		return TRUE
 	. = (!throwing) ? ..() : FALSE
 
+// We only do this for the extension as other stuff that overrides get_cell() handles EMP in an override.
```

Comment:
I'm not actually sure that any subtypes of /obj/item both override `emp_act()` and don't use this extension?

Reply:
`/obj/item/inducer/emp_act`